sshd.c

```diff
@@ -2602,7 +2598,15 @@ main(int ac, char **av)
 	io_sock_in = sock_in;
 	io_sock_out = sock_out;
 	if ((ssh = ssh_packet_set_connection(NULL, sock_in, sock_out)) == NULL)
+#ifdef WINDOWS
+	{
+		send_sshd_connection_telemetry(
+			"connection failed: unable to create connection");
+		fatal("Unable to create connection");
```

Comment:
You can remove the else block here and make the below line common for both windows and upstream.

fatal("Unable to create connection");

Reply:
In previous changes, I used the else block for cleaner upstream merges, since the if statement didn't have brackets (because it only had one line under it without the telemetry call). 

Should I still remove the else block?

Reply:
I got it. The current code is good.